integration/scenario.go

```diff
@@ -19,27 +19,29 @@ import (
 
 const scenarioHashLength = 6
 
-var errNoHeadscaleAvailable = errors.New("no headscale available")
-var errNoNamespaceAvailable = errors.New("no namespace available")
-var TailscaleVersions = []string{
-	"head",
-	"unstable",
-	"1.32.0",
-	"1.30.2",
-	"1.28.0",
-	"1.26.2",
-	"1.24.2",
-	"1.22.2",
-	"1.20.4",
-	"1.18.2",
-	"1.16.2",
-
-	// These versions seem to fail when fetching from apt.
-	// "1.14.6",
-	// "1.12.4",
-	// "1.10.2",
-	// "1.8.7",
-}
+var (
+	errNoHeadscaleAvailable = errors.New("no headscale available")
```

Comment:
why this is no const?

Reply:
Function calls can't be stored to const

Reply:
Argh. Sorry. Too tired. Read Error("foobar"), like in the regular code.